Solutions/Okta Single Sign-On/Hunting Queries/AdminPrivilegeGrant.yaml

```diff
@@ -1,6 +1,8 @@
 id: 5309ea6b-463c-4449-a3c4-2fc8ee0080ee
 name: Admin privilege granted (Okta)
 description: |
+  'Query checks for admin permissions granted to users/groups, often used by adversaries for access and privilege elevation. Verify known behavior and filter expected actions.'
+description-detailed: |
```

Comment:
This is not needed. 'Verify known behavior and filter expected actions.' Please rewrite the description.

Reply:
Updated description, please review it.